include/lsst/afw/formatters/Utils.h

```diff
@@ -75,50 +75,6 @@ bool extractOptionalFlag(std::shared_ptr<lsst::daf::base::PropertySet const> con
  */
 std::string const getItemName(std::shared_ptr<lsst::daf::base::PropertySet const> const& properties);
 
-/**
- * Returns the name of the table that a single slice of a pipeline involved in the processing
- * of a single visit should use for persistence of a particular output. All slices can be
- * configured to use the same (per-visit) table name using policy parameters.
- *
- * @param[in] policy   The `Policy` containing the table name pattern ("${itemName}.tableNamePattern",
- *                     where ${itemName} is looked up in `properties` using the "itemName" key)
- *                     from which the the actual table name is derived. This pattern may contain
- * a set of parameters in `%(key)` format - these are interpolated by looking up `"key"` in
- * the `properties` PropertySet.
- *
- * @param[in] properties   Provides runtime specific properties necessary to construct the
- *                         output table name.
- * @returns table name
- */
-std::string const getTableName(std::shared_ptr<lsst::pex::policy::Policy const> const& policy,
-                               std::shared_ptr<lsst::daf::base::PropertySet const> const& properties);
-
-/**
- * Stores the name of the table that each slice of a pipeline involved in processing a visit
- * used for persistence of its outputs. If slices were configured to all use the same (per-visit)
- * table name, a single name is stored.
- *
- * @param[in] policy   The `Policy` containing the table name pattern ("${itemName}.tableNamePattern",
- *                     where ${itemName} is looked up in `properties` using the "itemName" key)
- *                     from which the the actual table name is derived. This pattern may contain
- * a set of parameters in `%(key)` format - these are interpolated by looking up `"key"` in
- * the `properties` PropertySet.
- *
- * @param[in] properties   The runtime specific properties necessary to construct the table names.
- *
- * string. The `"visitId0"` property must also be present, and shall be a non-negative integer of type
- * `int64_t` uniquely identifying the current LSST visit. If the `"${itemName}.isPerSliceTable"`
- * property is present, is of type `bool` and is set to `true`, then it is assumed that
- * `"${itemName}.numSlices"` (a positive integer of type `int`) output tables exist and
- * are to be read in.
- *
- * @returns a list of table names
- * @see getTableName()
- */
-std::vector<std::string> getAllSliceTableNames(
-        std::shared_ptr<lsst::pex::policy::Policy const> const& policy,
-        std::shared_ptr<lsst::daf::base::PropertySet const> const& properties);
-
```

Comment:
Should the lines L55-57 also be removed?
```
namespace policy {
 class Policy;
```

Reply:
Yes, good catch!